src/components/Volume/index.tsx

```diff
@@ -32,7 +32,7 @@ const Volume = (props: VolumeProps) => {
       />
       <div className="volume-info">
         <div className="volume-header">
-          <h1><Link to={`books/${volume.id}`} target="__blank">{volume.title}</Link></h1>
+          <h1><Link to={`books/${volume.id}`} target="_blank">{volume.title}</Link></h1>
```

Comment:
I would add `rel="noreferrer noopener"` to limit potential security breaches

Reply:
Yes, right.